### PR TITLE
JSON

### LINKED ISSUE #1882: JS: JSON  Tutorial
### I will discuss what is JSON and how to use it?
```
Topics/8.JSON
```

### Domain

JavaScript

### Type of Contribution

- [x] Documentation
- [ ] Video
- [ ] Audio

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by yellowberard**
<hr>

## Description 📜
Brief description about json
Fixes #1882 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by yellowberard** [comment]
closes pr #8241

**Comment by yellowberard** [comment]
@2024-SANDHYA Please review my PR.

**Comment by 2024-SANDHYA** [comment]
More than half the content is plagiarised. 

image

**Comment by yellowberard** [comment]
I have written every thing on my own and not copied any thing.

**Comment by 2024-SANDHYA** [comment]
> I have written every thing on my own and not copied any thing.

Not possible !! If it's showing it means it is copied. Tryna rephrase the sentences then.

**Comment by yellowberard** [comment]
I am telling the truth I have not copied anything.

**Comment by yellowberard** [comment]
You can see the Every variable and string is written by me. It may be showing because of syntax section but I have rephrased that to also but I can't changed is completely as it is Syntax. @2024-SANDHYA

**Comment by 2024-SANDHYA** [comment]
> You can see the Every variable and string is written by me. It may be showing because of syntax section but I have rephrased that to also but I can't changed is completely as it is Syntax. @2024-SANDHYA

I haven't checked for the syntax, but the theory part, the content you wrote other than the code. Tryna rephrase the sentences that's the only thing.

**Comment by yellowberard** [comment]
I have made the changes where can I check for plagiarism. @2024-SANDHYA

**Review by 2024-SANDHYA — APPROVED** [review]

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Add reference links at the bottom @yellowberard

**Comment by yellowberard** [comment]
> Add reference links at the bottom @yellowberard

ok

**Comment by yellowberard** [comment]
> Add reference links at the bottom @yellowberard

I have made the changes @2024-SANDHYA

File: Web_Development/Javascript/JSON/JSON.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+## Why use JSON ??
+
+- Simplest format of data exchange i.e. easily understanable to a human .
+- Light weight i.e. easily data exchange.
+- Uses less data overall i.e. less cost for data exchange.
+- Multiple languages supported i.e. no repeated file for same data.
+
+## Acknowledgements
```
Comment (2024-SANDHYA) [inline Web_Development/Javascript/JSON/JSON.md:59]: Not the right word, write "references" instead. @yellowberard

**Comment by yellowberard** [comment]
I have made the changes @2024-SANDHYA.

**Pull request merged**